src/cata_tiles.cpp

```diff
@@ -1899,6 +1900,23 @@ bool cata_tiles::draw_from_id_string( std::string id, TILE_CATEGORY category,
         c ^= b;
         c -= rot32( b, 24 );
         loc_rand = c;
+
+        // idle tile animations:
+        if ( display_tile.animated ) {
+        	// idle animations run during the user's turn, and the animation speed
+        	// needs to be defined by the tileset to look good, so we use system clock:
+            auto now = std::chrono::system_clock::now();
+            auto now_ms = std::chrono::time_point_cast<std::chrono::milliseconds>(now);
```

Comment:
Astyle - Should be:

            auto now_ms = std::chrono::time_point_cast<std::chrono::milliseconds>( now );

Reply:
```suggestion
            auto now_ms = std::chrono::time_point_cast<std::chrono::milliseconds>( now );
```